app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -13,6 +19,16 @@ def notify_mas_tracking
     RrdMasNotificationMailer.build(self).deliver_now
   end
 
+  def send_rrd_pact_related_notification
+    icn = RapidReadyForDecision::ClaimContext.new(self).user_icn
+    client = Lighthouse::VeteransHealth::Client.new(icn)
```

Comment:
Actually, since this is a concern on the submission already, does this work?
```suggestion
    client = Lighthouse::VeteransHealth::Client.new(user_icn)
```

Reply:
Nevermind, I got confused.